src/main/java/org/jenkins/plugins/lockableresources/LockStepResource.java

```diff
@@ -111,6 +115,9 @@ public static FormValidation doCheckLabel(@QueryParameter String value, @QueryPa
 			if ((resourceLabel == null) && (resourceName == null)) {
 				return FormValidation.error("Either label or resource name must be specified.");
 			}
+			if (resourceLabel != null && !LockableResourcesManager.get().isValidLabel(resourceLabel)) {
```

Comment:
I'd add this inside the above if block, so the duplicate resourceLabel != null isn't needed, slightly more readable imho

Reply:
Not sure about which "above if block" you're talking about. The one just above is clearly using a completely different condition, so I guess you're talking about this one:

```
 			if (resourceLabel != null && resourceName != null) {
 				return FormValidation.error("Label and resource name cannot be specified simultaneously.");
 			}
```

So you'd want me to replace it with this?


```
 			if (resourceLabel != null) {
 				if (resourceName != null) {
 					return FormValidation.error("Label and resource name cannot be specified simultaneously.");
 				}
				if (!LockableResourcesManager.get().isValidLabel(resourceLabel)) {
					return FormValidation.error("The label does not exist: " + resourceLabel);
				}
 			}
```

Not sure it's really more readable, but sure I can do that. Care to confirm?